### ISSUE TITLE
Release 0.15.0

### ISSUE THREAD

**Issue by selaux** [ja2-stracciatella/ja2-stracciatella#312]
Roadmap for the Release 0.15.0. What still needs to be done.

- [x] Update and merge changes up until now (#241)
- [x] Update contributors
- [x] Update Version number to `0.15.0`
- [x] Github prerelease and tags
- [x] Attach built versions to release
  - [x] Linux, Windows
  - [x] OS X
- [x] Runtime tests of release packages
 - [x] Linux
    - [x] Ubuntu 16.04 x64
    - [x] Ubuntu 14.04 i386
 - [x] Windows
 - [x] OS X
- [x] Make prerelase the release
- [x] Announce
  - [x] Website
  - [x] Bears pit
  - [x] Moddb
- [x] Reset version to development `0.15.x`
- [x] Update and merge release process information (#284)

**Comment by selaux** [comment]
Feel free to test the files attached to the release. I will build for OS X tomorrow.

**Comment by lynxlynxlynx** [comment]
I've tested the 64bit deb and all seems fine. I didn't capture Drassen though, so I didn't test Bobby Ray's in a fresh game.

**Comment by JAsmine-ja2** [comment]
I noticed some minor stuff #313

**Comment by JAsmine-ja2** [comment]
I tested v0.15 windows and it seems to work! :)
- creating IMP
- hiring from AIM and MERC
- taking Omerta and Drassen
- talking to rebels and recruiting Ira
- ordering from BR
- mails and RIS report look fine, too.

**Comment by selaux** [comment]
I also tested Windows, and found #317. This should not be a release breaking issue though imho.

**Comment by lynxlynxlynx** [comment]
I wrote a draft announcement for BP:

> JA2-Stracciatella 0.15.0
> 
> The JA2-Stracciatella team is proud to announce a new release, after the project was resurrected a few months ago [http://ja2-stracciatella.github.io/news/]. It includes several bugfixes and some smaller improvements, besides the usual code cleanup. Larger changes have been postponed to the next version, so expect a more interesting changelog in the months to come.
> 
> You can follow the project at its new web site: http://ja2-stracciatella.github.io/
> 
> copy and tag changelog from
> https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/changes.md

**Comment by lynxlynxlynx** [comment]
@selaux agreed.

**Comment by selaux** [comment]
@lynxlynxlynx Sounds good. In parallel I've done the same for the homepage: https://github.com/ja2-stracciatella/ja2-stracciatella.github.io/blob/release-0.15.0/_posts/2016-05-23-release-0.15.0.md

**Comment by selaux** [comment]
Tested on Ubuntu 14.04 i386 up until recruiting Ira.

**Comment by selaux** [comment]
I updated the release packages (except os x). Ubuntu i386 still works so that is a good indicator :smile:.

**Comment by derSteFfi** [comment]
tested on arch linux too.
what about the version number in main menu/loading screen?
and what does 'Make prerelase the release' mean? :)

**Comment by lynxlynxlynx** [comment]
The version number was already 0.15.0 in the x64 build. The prerelease thing is a github "label", most noticeable if you're using its api.

**Comment by JAsmine-ja2** [comment]
While anouncing the release on BP, you could maybe pm Kermi, so his famous FTP repository references Stracciatella continued?

http://kermi.pp.fi/JA_2/Stracciatella/

**Comment by lynxlynxlynx** [comment]
Are we waiting for something else?

**Comment by selaux** [comment]
So I built and tested on OS X without major issues except for the performance lacking quite a lot (#319). The game is still playable though so this shouldn't be a release blocker.

**Comment by lynxlynxlynx** [comment]
Yeah, let's ship it, only larger resolutions are affected.

**Comment by lynxlynxlynx** [comment]
Ok, most of the announcement work is done. 
@JAsmine-ja2 can you update the moddb entry please?

**Comment by JAsmine-ja2** [comment]
Mod DB entry is updated. Article needs to be authorised and might take some time to show. New versions are uploaded. Changelog is there, too.

**Comment by lynxlynxlynx** [comment]
Now only #320 and #284 remain. :)

**Comment by selaux** [comment]
Merged last PR :confetti_ball:. Thanks to everyone participating in the release. Nice job :+1:

**Comment by lynxlynxlynx** [comment]
Good job team! :cake:

**Comment by derSteFfi** [comment]
yes, good job (at) all!
